microk8s-resources/actions/enable.gpu.sh

```diff
@@ -13,12 +13,11 @@ else
   exit 1
 fi
 
-echo "Enabling DNS"
-/snap/bin/microk8s.enable dns
-
 refresh_opt_in_config "default-runtime" "nvidia" dockerd
 sudo systemctl restart snap.${SNAP_NAME}.daemon-docker
-sleep 5
+sleep 10
```

Comment:
Could we instead poll the status of the daemon until it's started, or we hit a reasonable timeout? Just setting it to 10s seems bad.